src/main/resources/org/datadog/jmxfetch/default-jmx-metrics.yaml

```diff
@@ -0,0 +1,145 @@
+# Memory
+- include:
+    domain: java.lang
+    type: Memory
+    attribute:
+      HeapMemoryUsage.used:
+        alias: jvm.heap_memory
+        metric_type: gauge
+      HeapMemoryUsage.committed:
+        alias: jvm.heap_memory_committed
+        metric_type: gauge
+      HeapMemoryUsage.init:
+        alias: jvm.heap_memory_init
+        metric_type: gauge
+      HeapMemoryUsage.max:
+        alias: jvm.heap_memory_max
+        metric_type: gauge
+      NonHeapMemoryUsage.used:
+        alias: jvm.non_heap_memory
+        metric_type: gauge
+      NonHeapMemoryUsage.committed:
+        alias: jvm.non_heap_memory_committed
+        metric_type: gauge
+      NonHeapMemoryUsage.init:
+        alias: jvm.non_heap_memory_init
+        metric_type: gauge
+      NonHeapMemoryUsage.max:
+        alias: jvm.non_heap_memory_max
+        metric_type: gauge
+
+# Threads
+- include:
+    domain: java.lang
+    type: Threading
+    attribute:
+      ThreadCount:
+        alias: jvm.thread.count
+        metric_type: gauge
+      DaemonThreadCount:
+        alias: jvm.thread.daemon_count
+        metric_type: gauge
+
+# CPU load
+- include:
+    domain: java.lang
+    type: OperatingSystem
+    attribute:
+      ProcessCpuLoad:
+        alias: jvm.cpu_load.process
+        metric_type: gauge
+      SystemCpuLoad:
+        alias: jvm.cpu_load.system
+        metric_type: gauge
+
+# Young Gen Collectors (Minor Collections)
+- include:
+    domain: java.lang
+    type: GarbageCollector
+    name: Copy
```

Comment:
Not seeing a mention to this filter in https://docs.datadoghq.com/integrations/java/#description-of-the-filters. What does it do ? And how do we differenciate `jvm.gc.minor_collection_count` from `Copy` vs `ParNew` for ex ?

Reply:
Thad documentation says `On top of these parameters, the filters support “custom” keys which means that you can filter by bean parameters. ` - which is what we are using here. We are just filtering by specific GC name.

Reply:
Okay I understand now, seems like I missed that part 😅 . Now for the second part of my comment, I guess you can only have one `Young Gen Collector` and one `Old Gen Collector` ?

Reply:
Yes, JVM allows only one Old and one New gen collector - by there are a few different possibly types of those.